components/Tabs/src/private/MDCItemBar.m

```diff
@@ -275,13 +284,15 @@ - (void)layoutSubviews {
   _collectionView.frame = bounds;
 
   // Update collection metrics if the size has changed.
-  if (!CGSizeEqualToSize(bounds.size, _lastSize)) {
+  if (!CGSizeEqualToSize(bounds.size, _lastSize) ||
+      self.adjustedCollectionViewWidth != _lastAdjustedCollectionViewWidth) {
```

Comment:
Nit: As a method, this should be invoked using the "brackets" syntax `[self adjustedCollectionViewWidth]`.